src/components/Layout/Header/Header.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { MemoryRouter } from 'react-router-dom';
+import Header from './Header';
+
+const meta = {
+  title: 'Components/Layout/Header',
+  component: Header,
+} satisfies Meta<typeof Header>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Index: Story = {
+  decorators: [
+    (Story) => (
+      <MemoryRouter initialEntries={['/']}>
+        <Story />
+      </MemoryRouter>
+    ),
+  ],
+};
+
+export const Regular: Story = {
+  decorators: [
+    (Story) => (
+      <MemoryRouter initialEntries={['/index']}>
```

Comment:
There is no valid URL called `/index`, would be better to use the URL that actually exists, even if it works this way.